cg/meta/workflow/fluffy.py

```diff
@@ -2,24 +2,52 @@
 import logging
 import shutil
 from pathlib import Path
-from typing import List, Optional
+from typing import Dict, List, Optional
 
 import pandas as pd
 from sqlalchemy.orm import Query
 from cg.constants import Pipeline
 from cg.constants.constants import FileFormat
-from cg.constants.demultiplexing import SampleSheetBcl2FastqSections
+from cg.constants.demultiplexing import SampleSheetBCLConvertSections, SampleSheetBcl2FastqSections
 from cg.exc import HousekeeperFileMissingError
 from cg.io.controller import ReadFile
 from cg.meta.demultiplex.housekeeper_storage_functions import get_sample_sheets_from_latest_version
 from cg.meta.workflow.analysis import AnalysisAPI
 from cg.models.cg_config import CGConfig
+from cg.models.fluffy import FluffySampleSheet, FluffySampleSheetEntry, FluffySampleSheetHeader
 from cg.store.models import Family, Flowcell, Sample
 from cg.utils import Process
+from cg.io.sample_sheet import read_sample_sheet
 
 LOG = logging.getLogger(__name__)
 
 
+def get_data_header_key(sample_sheet: Dict[str, List[str]]) -> str:
+    sample_sheet_headers: List[List[str]] = [
+        SampleSheetBCLConvertSections.Data.HEADER.value,
+        SampleSheetBcl2FastqSections.Data.HEADER.value,
```

Comment:
I think this is what you want
```suggestion
        SampleSheetBCLConvertSections.Data.COLUMNS.value,
        SampleSheetBcl2FastqSections.Data.COLUMNS.value,
```